upstream/solr6/local/92-init-solr.sh

```diff
@@ -109,6 +109,17 @@ function makeConfigs {
         setOption 'alfresco.suggestable.property.2' '{http://www.alfresco.org/model/content/1.0}description' "$SHARED_PROPERTIES"
         setOption 'alfresco.suggestable.property.3' '{http://www.alfresco.org/model/content/1.0}content' "$SHARED_PROPERTIES"
     fi
+
```

Comment:
maybe we should refactor `$ALFRESCO_SOLR_SUGGESTER_ENABLED`into `$SHARED_ALFRESCO_SOLR_SUGGESTER_ENABLED` since it is also concerns the shared.properties file?

Reply:
This environment variable is named that way because of backwards compatibility: older versions of solr used to have a single property to enable the suggester: https://github.com/xenit-eu/docker-solr/blob/7414ea341091795a68bc92dedd345641c0c1b920/upstream/solr4/local/92-init-solr.sh#L87

Additionally, changing that environment variable _now_ would break backwards compatibility with previous solr6 images, which we should not do.